src/pages/PageCallManage/PageCallManage.tsx

```diff
@@ -0,0 +1,272 @@
+  }
+
+  @action toggleButtons = () => {
+    this.showButtonsInVideoCall = !this.showButtonsInVideoCall
+  }
+  @action hideButtonsIfVideo = () => {
+    if (
+      !this.props.isFromCallBar &&
+      !this.alreadySetShowButtonsInVideoCall &&
+      callStore.currentCall?.remoteVideoEnabled
+    ) {
+      this.showButtonsInVideoCall = false
+      this.alreadySetShowButtonsInVideoCall = true
+    }
+  }
+
+  renderCallTime = () => {
+    const timeInSec =
+      moment(this.state.curTime).diff(this.state.startedTime) / 1000
+    const isSecs = toInteger(timeInSec % 6)
+    const secs = isSecs < 10 ? '0' + isSecs : isSecs
+    const isMins = toInteger(timeInSec / 6)
+    const mins = isMins < 10 ? '0' + isMins : isMins
+    const isHours = toInteger(timeInSec / 60)
+    const hours = isHours ? isHours + ':' : null
+
+    return (
+      <View style={styles.timerContainer}>
+        <View style={styles.timerDisplayBox}>
+          <RnText style={{ color: 'white' }}>
+            {hours}
+            {mins}:{secs}
+          </RnText>
+        </View>
+      </View>
+    )
+  }
+
+  renderCall = (c?: Call, isVideoEnabled?: boolean) => (
+    <Layout
+      compact
+      dropdown={
+        isVideoEnabled && !c?.transferring
+          ? [
+              {
+                label: this.showButtonsInVideoCall
+                  ? intl`Hide call menu buttons`
+                  : intl`Show call menu buttons`,
+                onPress: this.toggleButtons,
+              },
+            ]
+          : undefined
+      }
+      noScroll
+      onBack={Nav().backToPageCallRecents}
+      title={c?.partyNumber || intl`Connection failed`}
+      transparent={!c?.transferring}
+    >
+      {!c ? null : c.transferring ? (
+        <PageTransferAttend />
+      ) : (
+        <>
+          {isVideoEnabled && this.renderVideo(c)}
+
+          {!isVideoEnabled && (
+            <CallerInfo
+              isUserCalling={!c.partyNumber.includes('+')}
+              callerName={c.getCallerName}
+              callerNumber={c.partyNumber}
+              containerStyle={{ marginTop: c.getCallerName ? '5%' : '20%' }}
+            />
+          )}
+          {this.renderCallTime()}
+          {this.renderBtns(c, isVideoEnabled)}
+          {this.renderHangupBtn(c, isVideoEnabled)}
+        </>
+      )}
+    </Layout>
+  )
+
+  renderVideo = (c: Call) => (
+    <>
+      <View style={styles.videoSpace} />
+      <View style={styles.video}>
+        <VideoPlayer sourceObject={c.remoteVideoStreamObject} />
+      </View>
+      <RnTouchableOpacity
+        onPress={this.toggleButtons}
+        style={StyleSheet.absoluteFill}
+      />
+    </>
+  )
+
+  renderBtns = (c: Call, isVideoEnabled?: boolean) => {
+    if (isVideoEnabled && !this.showButtonsInVideoCall) {
+      return null
+    }
+    const Container = isVideoEnabled ? RnTouchableOpacity : View
+    const activeColor = CustomColors.IconActiveBlue
+    const n = callStore.backgroundCalls.length
```

Comment:
add meaningful name to a variable. I am not able to undestand what is the meaning of `n`

Reply:
In this part of the code, I haven't made any changes Jigar.  Since I moved this file to a folder, it is showing as new changes. variable n is existing code in the project. Anyhow, I am modifying this page for the next ticket. It will be fixed in the next pr.